girder/utility/gridfs_assetstore_adapter.py

```diff
@@ -65,6 +65,7 @@ def validateInfo(doc):
                 ('uuid', pymongo.ASCENDING),
                 ('n', pymongo.ASCENDING)
             ], unique=True)
+            chunkColl.create_index([('n', pymongo.ASCENDING)])
```

Comment:
Hm, that's odd. Any idea why that query wouldn't use the compound index? I would expect it to do so...